ivy/functional/ivy/layers.py

```diff
@@ -697,16 +697,27 @@ def conv2d(
               [[2.],[4.],[6.]]
               ]])
 
-    With :code:`ivy.NativeArray` input:
-
-    >>> x = ivy.native_array(ivy.random_normal(mean=0, std=1, shape=[1, 32, 32, 3]))
-    >>> filters = ivy.native_array(ivy.random_normal(mean=0, std=1, \
-                                   shape=[3, 5, 3, 5])) #HWIO
-    >>> result = ivy.conv2d(x, filters, [2, 1], 'VALID') \
-        #non-square filter with unequal stride and valid padding
-    >>> print(result.shape)
-    (1, 15, 28, 5)
+    With one :code:`ivy.Container` input:
+    >>> x = ivy.Container(a=ivy.)
+    >>> filters = ivy.
+    >>> result = ivy.conv2d(x, filters, (2,), 'SAME')
+    >>> print(result)
+    {
+        a:ivy.array([])
+    }
 
+    With multiple :code:`ivy.Container` inputs:
+    >>> x = ivy.Container(a = ivy.,  \
```

Comment:
Invalid syntax, you should be creating an array using `ivy.array` instead of just `ivy.`